### PR TITLE
[Subs 1559] submission submittable summary

### PR DESCRIPTION
**Pull request by ke4**

### CODE REVIEW

File: src/main/java/uk/ac/ebi/subs/api/controllers/SubmissionContentsIssuesSummaryController.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    @NonNull
+    private FileRepository fileRepository;
+    @NonNull
+    private ValidatorResultRepositoryCustom validatorResultRepositoryCustom;
+    @NonNull
+    private ValidationResultRepository validationResultRepository;
+    @NonNull
+    private ProcessingStatusRepository processingStatusRepository;
+
+    @GetMapping(value = "/submissions/{submissionId}/contents/issuesSummary")
```
Comment (davidrichardson) [inline src/main/java/uk/ac/ebi/subs/api/controllers/SubmissionContentsIssuesSummaryController.java]: can we give this a more specific name + url?

How about submission blockers ?

File: src/main/java/uk/ac/ebi/subs/api/controllers/SubmissionContentsIssuesSummaryController.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    public SubmissionIssuesSummary getSubmissionContentsIssuesSummary(@PathVariable @P("submissionId") String submissionId) {
```
Comment (davidrichardson) [inline src/main/java/uk/ac/ebi/subs/api/controllers/SubmissionContentsIssuesSummaryController.java]: Can you wrap this in a Resource and add a self link?

File: src/main/java/uk/ac/ebi/subs/api/controllers/SubmissionContentsIssuesSummaryController.java (hunk 1 above)
Comment (ke4) [reply src/main/java/uk/ac/ebi/subs/api/controllers/SubmissionContentsIssuesSummaryController.java]: OK, I will change it

File: src/main/java/uk/ac/ebi/subs/api/controllers/SubmissionContentsIssuesSummaryController.java (hunk 2 above)
Comment (ke4) [reply src/main/java/uk/ac/ebi/subs/api/controllers/SubmissionContentsIssuesSummaryController.java]: OK. I will do it.

**Comment by davidrichardson** [comment]
Question - is a link to this controller added to the submission or submission contents links?

**Pull request merged**